.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,15 +12,13 @@ medic/cht-core#[number]
 - [ ] Internationalised: All user facing text
 - [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
-# Compose URLs:
-<!-- After CI passes, PR submitter should manually replace these placeholders with deep links to staging. Makes for easier testing! -->
-<!-- e.g. https://staging.dev.medicmobile.org/_couch/builds/medic:medic:<branch>/docker-compose/cht-core.yml  -->
-<!-- e.g. https://staging.dev.medicmobile.org/_couch/builds/medic:medic:<branch>/docker-compose/cht-couchdb.yml   -->
-<!-- e.g. https://staging.dev.medicmobile.org/_couch/builds/medic:medic:<branch>/docker-compose/cht-couchdb-clustered.yml  -->
+# Compose URLs
+<!-- Do not change these!  CI will automatically update these to be the deep URLs -->
+If Build CI hasn't passed, these may 404:
```

Comment:
Why not trigger the replacement at the end of the PR build, so it only replaces if successfully published?

Reply:
Honestly? It was because I was like 30+ commits in to just get it working as is and I got tired.  If you think it's too broken as is, I can go back to try and fix it. For now,  I'll push back a lil and see if you're OK with it ;)